src/sentry/static/sentry/app/components/idBadge/projectBadge.tsx

```diff
@@ -3,15 +3,25 @@ import PropTypes from 'prop-types';
 
 import BaseBadge from 'app/components/idBadge/baseBadge';
 import BadgeDisplayName from 'app/components/idBadge/badgeDisplayName';
+import {Project} from 'app/types';
 
-export default class ProjectBadge extends React.PureComponent {
+type Props = {
+  project: Project;
+  avatarSize: number;
+  className?: string;
+  hideAvatar?: boolean;
+  hideName?: boolean;
+  description?: React.ReactNode;
+  avatarClassName?: string | React.CSSProperties;
```

Comment:
When would this be `CSSProperties`?

Reply:
I thought was the right type to use for style attributes based on `BaseBadge` having a proptype of `string, object` and the resulting prop ending up in a `css` prop of `StyledAvatar`.